azure/services/virtualmachineimages/cache.go

```diff
@@ -119,7 +119,7 @@ func (c *Cache) Get(ctx context.Context, location, publisher, offer, sku string)
 			return compute.ListVirtualMachineImageResource{}, err
 		}
 	} else {
-		log.Info("VM images cache hit", "location", key.location, "publisher", key.publisher, "offer", key.offer, "sku", key.sku)
+		log.V(4).Info("VM images cache hit", "location", key.location, "publisher", key.publisher, "offer", key.offer, "sku", key.sku)
```

Comment:
In case you're wondering, see https://kubernetes.slack.com/archives/CEX9HENG7/p1659637807401169?thread_ts=1659637507.302629&cid=CEX9HENG7 for why I changed this.